src/core/types/modules.d.ts

```diff
@@ -67,5 +67,10 @@ declare namespace NodeJS {
 
 		SLACK_TOKEN: string | undefined;
 		SLACK_CHANNEL: string | undefined;
+
```

Comment:
maybe add comments like with the others, just to keep it clean for anyone to go through the list